### PR TITLE
[WIP] Integrate db using mongoose

### PR DESCRIPTION
**Pull request by mytoykar**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -1,12 +1,35 @@
 const express = require('express');
+const mongoose = require('mongoose');
 
 const app = express();
-
 const port = 6969;
+const dbuser = 'user';
+const dbpassword = 'password';
```
Comment (carlogino) [inline backend/app.js:7]: mas okay siguro boss kung may config file?

File: backend/app.js [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
 app.listen(port, () => {
   console.log('listen port 6969');
 });
 
 app.get('/hello_world', (req, res) => {
   res.send('yametekudasai neechan');
+  console.log('called hello world at this time: ' + new Date());
+  var accountsModel = new AccountsModel();
+  accountsModel.account_id = 10;
+  accountsModel.limit = 6969;
```
Comment (carlogino) [inline backend/app.js:32]: same din sa mag eto

File: backend/app.js [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
 
 const app = express();
-
 const port = 6969;
+const dbuser = 'user';
+const dbpassword = 'password';
+const analytics = 'sample_analytics';
+var mongoUrl = `mongodb+srv://${dbuser}:${dbpassword}@aws-mongodb-cluster0-lvdqf.mongodb.net/${analytics}?retryWrites=true&w=majority`;
+
+mongoose.connect(mongoUrl, { useNewUrlParser: true });
```
Comment (hailwaft) [inline backend/app.js:11]: pa add din nito `useUnifiedTopology: true` kasi may deprecation warning na lumalabas dyan

Comment (carlogino) [reply backend/app.js:11]: Nice

**Pull request closed without merging**